### ISSUE TITLE
Unit tests are broken (missing implementation Powon\Dao\MemberDAO::deleteMember in the stub)

### ISSUE THREAD

**Issue by xmithd** [avengers5531/db_powon#37]
```
....PHP Fatal error:  Class Powon\Test\Stub\MemberDaoStub contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Powon\Dao\MemberDAO::deleteMember) in /var/www/tests/Stub/MemberDAOStub.php on line 138
```

**Comment by xmithd** [comment]
I'm writing tests for posts and I'll fix the tests. The failing tests indicate there is actually a bug with member update.

**Comment by xmithd** [comment]
Fixed in changeset 91e806a787e4b136000572ccae7c928458f3cb67
